tutorials/Tutorial_Gallery.ipynb

```diff
@@ -57,7 +57,8 @@
     "- [Quantile Loss](./Quantile_Loss.ipynb)\n",
     "- [Murphy Diagrams](./Murphy_Diagrams.ipynb)\n",
     "- [Flip-Flop Index](./Flip_Flop_Index.ipynb)\n",
-    "- [Consistent Scores](./Consistent_Scores.ipynb)"
+    "- [Consistent Scores](./Consistent_Scores.ipynb)\n",
+    "- [Threshold Weighted Scores](./Threshold_Weights_Scores.ipynb)"
```

Comment:
I think this should link to Threshold_Weighted_Scores.ipynb

Reply:
Yes you're right. I'll update it